app/templates/signed-out.html

```diff
@@ -5,11 +5,11 @@
 {% block main %}
 
   <div class="u-mb-s">
-    <h1 class="saturn">Your survey has been saved</h1>
+    <h1 class="saturn">Your survey has been saved. You are <br> now signed out.</h1>
```

Comment:
Why do we have an explicit break in this H1?

Reply:
To match design on screenshots on the card i.e. having the text on two lines: https://trello.com/c/7VACxFOq/1016-clear-actions-to-exit-or-continue-a-survey-s

Reply:
I think we should be relying more on the browser wrapping the text rather then putting explicit breaks in?

Reply:
@jonnyshaw89 I agree

Reply:
Just to illustrate the point of why you should be judicious about using line-breaks:

<img width="250" alt="screenshot 2017-10-05 13 31 57" src="https://user-images.githubusercontent.com/930398/31227426-ba8218dc-a9d1-11e7-84cc-66c77aa8096d.png">